ts/src/profiler.ts

```diff
@@ -102,7 +118,7 @@ function isRequestProfile(prof: any): prof is RequestProfile {
 }
 
 /**
- * Returns true if response has statusCode.
+ * @return - true if response has statusCode and false otherwise.
```

Comment:
You could just say "true iff the response has the status field.", that implies the false behavior.

Do not use " - " after @return. So just "@return true iff response has the status field." Please fix the style in other places.

Reply:
I should use @return everywhere. 
I've fixed this everywhere in PR #78, and am changing in profiler.ts here.